dss/events/handlers/index.py

```diff
@@ -52,19 +67,18 @@ def is_bundle_to_index(key) -> bool:
     return result is not None
 
 
-def read_bundle_manifest(s3, bucket_name, bundle_key, logger):
-    manifest_string = s3.Object(bucket_name, bundle_key).get()['Body'].read().decode("utf-8")
+def read_bundle_manifest(blobstore, bucket_name, bundle_key, logger):
```

Comment:
https://github.com/HumanCellAtlas/data-store/issues/390

Reply:
This will be a separate PR